lib/api/src/js/instance.rs

```diff
@@ -173,6 +173,12 @@ impl Instance {
     pub fn store(&self) -> &Store {
         self.module.store()
     }
+
+    /// Returns the inner WebAssembly Instance
+    #[doc(hidden)]
```

Comment:
Is this API meant to be publicly usable? In that case it shouldn't be hidden from the documentation.

Reply:
Because it differs from the SYS API, I rather keep it hidden so people don't rely on it.

Although, we need it for wasmer-js